console/network/src/testnet_v0.rs

```diff
@@ -136,13 +136,17 @@ impl Network for TestnetV0 {
     /// A list of (consensus_version, block_height) pairs indicating when each consensus version takes effect.
     /// Documentation for what is changed at each version can be found in `N::CONSENSUS_VERSION`
     #[cfg(not(any(test, feature = "test")))]
-    const CONSENSUS_VERSION_HEIGHTS: [(ConsensusVersion, u32); 3] =
-        [(ConsensusVersion::V1, 0), (ConsensusVersion::V2, 2_950_000), (ConsensusVersion::V3, 4_800_000)];
+    const CONSENSUS_VERSION_HEIGHTS: [(ConsensusVersion, u32); 4] = [
+        (ConsensusVersion::V1, 0),
+        (ConsensusVersion::V2, 2_950_000),
+        (ConsensusVersion::V3, 4_800_000),
+        (ConsensusVersion::V4, 5_000_000),
```

Comment:
Note: These values will need to be updated once the code logic is complete

Reply:
Updated in [c931eae](https://github.com/ProvableHQ/snarkVM/pull/2601/commits/c931eae9df1920606a1fcfd7c7d1a9cc0b7aed03) and will double-check on closer to the activation dates 👍